CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## [1.0.1] - [May 31, 2022]
+* Add maxWidth to StatusAlert.show
```

Comment:
You know that this is what people will read while checking what's new in this lib and your "Add maxWidth to StatusAlert.show" doesn't make any sense. What you have to write in the changelog is something clear and understandable, e.g. "new maxWidth parameter will let you make the Status Alert smaller on big screens"

Reply:
Fixed.